src/task1.java

```diff
@@ -0,0 +1,9 @@
+
+public class task1 {
+    public static void main(String...arg) {
+        int x = 7;
+        int y = 5;
+        float b = (float) x / y;
+            System.out.println(x + "/" + y + "=" + b);
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"